### PR TITLE
[pt] Update README.pt.md

### PR DESCRIPTION
**Pull request by horstmannmat**
Update #1318 

Signed-off-by: Matheus Horstmann <mch15@inf.ufpr.br>

### CODE REVIEW

**Comment by RockyTV** [comment]
I was discussing this on [Mastodon](https://mastodon.technology/@xinayder/101111711348509115) and after reading a bit on the i18n issue, I have some feedback:

- You switch from `se você deseja apoiar` to `se desejas apoiar`?
- Throughout the text, you use both `você` and `tu`
- There are some minor mistakes such as using `não exite`, which should be `não hesite`
 - `Cada lançamento é assinado [...] e publicadas` should be `publicado`, and a personal suggestion, I think `versão` instead of `lançamento` fits better

The way I see it, it looks like if you mixed both `pt_PT` and `pt_BR`. It's not bad, but there are a few inconsistencies which made me think that.

File: README.pt.md [hunk 1]
```diff
@@ -1,94 +1,99 @@
...
-o recurso especificado, e você pode criar uma premiação para qualquer recurso que desejar.
-O Patreon é melhor direcionado para a manutenção a longo prazo do Sway.
+Se desejas apoiar o desenvolvimento do projeto, podes contribuir com
+SirCmpwn em sua [página no Patreon][sircmpwn patreon].
 
 ## Ajuda em português
 
-No momento, o suporte em português no canal do IRC **não está ativo**. Em caso de problemas,
-use as [*issues*](https://github.com/swaywm/sway/issues/) do Github (*em inglês*).
+No momento, o suporte em português no canal no IRC **não está ativo**.
```
Comment (bemeurer) [inline README.pt.md]: I think the second `no` is redundant, `canal IRC` should suffice,

File: README.pt.md [hunk 2]
```diff
@@ -1,94 +1,99 @@
...
-ou queira colaborar com a tradução, visite
-[essa *issue*](https://github.com/swaywm/sway/issues/1318) para mais informações e não
-exite em enviar quaisquer correções necessárias.
+A tradução em português é um **trabalho em progresso**. Caso encontre
+erros ou queira colaborar com a tradução, visite [essa *issue*][sway i18n]
+para mais informações e não hesite em enviar quaisquer correções
+necessárias.
 
-## Assinaturas dos *Releases*
+## Assinaturas dos lançamentos
```
Comment (bemeurer) [inline README.pt.md]: I don't think `lançamentos` is usual terminology. If anything I'd stick with the anglicized `Releases`.

File: README.pt.md [hunk 3]
```diff
@@ -1,94 +1,99 @@
...
 
-*Releases* são assinadas com a chave
-[B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
-e publicadas [no GitHub](https://github.com/swaywm/sway/releases).
+Cada versão é assinada com a chave [B22DA89A][sway pgp] e publicada
+[no Github][sway releases].
 
 ## Instalação
 
 ### A partir de pacotes
```
Comment (bemeurer) [inline README.pt.md]: I  think `Por` is better than `A partir de` for this and subsequent subsections.

File: README.pt.md [hunk 4]
```diff
@@ -1,94 +1,99 @@
...
-procure por informações sobre como instalar o Sway na sua distribuição
-[aqui](https://github.com/swaywm/sway/wiki/Unsupported-packages).
+Sway está disponível em várias distribuições. Verifique se o pacote
+"sway" está disponível para ser instalado na sua distribuição GNU/Linux. Caso não esteja,
+procure por informações sobre como instalar Sway na sua distribuição
+[aqui][sway pkg wiki].
 
-Se você está interessado em manter um pacote do Sway para a sua distribuição,
-visite o canal no IRC ou mande um email para sir@cmpwn.com (*em inglês*).
+Se estiveres interessado em manter um pacote em sua distribuição, visite o
```
Comment (bemeurer) [inline README.pt.md]: `você estiver`

File: README.pt.md [hunk 5]
```diff
@@ -1,94 +1,99 @@
...
+canal no IRC ou mande um e-mail para <sir@cmpwn.com> (*em inglês*).
```
Comment (bemeurer) [inline README.pt.md]: Add a link to the IRC channel.

Comment (bemeurer) [inline README.pt.md]: `envie um e-mail` as opposed to `mande um e-mail`

File: README.pt.md [hunk 6]
```diff
@@ -1,94 +1,99 @@
...
 
+Sway removerá as permissões de root logo após o início do processo.
+
 ## Configuração
 
-Se você já usa o i3, copie o arquivo de configuração do i3 para `~/.config/sway/config`;
-o Sway lerá o arquivo normalmente. Senão, copie o arquivo de configuração de exemplo
-para `~/.config/sway/config`. É comum esse arquivo estar localizado em
-`/etc/sway/config`. Veja `man 5 sway` para informações sobre configuração.
+Se já usas i3, copie teu arquivo de configuração para `~/.config/sway/config`
```
Comment (bemeurer) [inline README.pt.md]: `Se você já usa i3, copie sua configuração para ...`

Comment (bemeurer) [reply README.pt.md]: I think `configuração` already strongly implies that it's an `arquivo` so it becomes redundant.

File: README.pt.md [hunk 7]
```diff
@@ -1,94 +1,99 @@
...
+e Sway lerá o arquivo normalmente.  Senão, copie o arquivo de configuração
```
Comment (bemeurer) [inline README.pt.md]: I think `usará` is better than `lerá` here.

Comment (bemeurer) [inline README.pt.md]: I believe the correct form is `Se não`, no?

File: README.pt.md [hunk 8]
```diff
@@ -1,94 +1,99 @@
...
+de exemplo para `~/.config/sway/config`. É comum esse arquivo estar localizado
```
Comment (bemeurer) [inline README.pt.md]: `arquivo de configuração exemplo`, the `de` seems unnecessary.

File: README.pt.md [hunk 9]
```diff
@@ -1,94 +1,99 @@
...
+em `/etc/sway/config`. Veja `man 5 sway` para informações sobre configuração.
 
 ## Executando
 
-Execute `sway` a partir de um terminal do Linux. Alguns gerenciadores de *display*
-podem funcionar, porém o Sway não procura manter compatibilidade com esses (segundo
-relatos, o gdm funciona bem com o Sway).
-
+Execute `sway` a partir do console. Alguns gerenciadores de *display* podem
+funcionar, porém não são oficialmente suportados (segundo relatos, gdm
```
Comment (bemeurer) [inline README.pt.md]: Maybe link to gdm.

**Comment by bemeurer** [comment]
Just an addendum after having review the file, this is through and through a Brazilian Portuguese doc, if someone wishes to add a Portugal Portuguese README it should have some changes.

**Comment by horstmannmat** [comment]
> Having lived in Brazil and Portugal, I think I qualify as a speaker of both `pt_PT` and `pt_BR`. Some thoughts:
> 
>     1. It's not worth it, IMO, to hybridize the dialects, this will lead to more confusion that anything.
>
As I said in [Mastodon](https://mastodon.technology/@horstmannmat/101112187770806353) European Portuguese is understandable by Brazilians, and vice-versa. And translate for both at same time could sound strange.

**Comment by bemeurer** [comment]
> translate for both at same time could sound strange.

Yes, that's what I said, translating to both at the same time would sound strange.

> European Portuguese is understandable by Brazilians, and vice-versa.

Sure, this is mostly true with some minor exceptions. This doesn't, however, negate the fact that the dialects are different enough that it can feel uncomfortable for a native speaker of one to read a document in the other, specially a technical one.

I think we are fine with just a Brazilian Portuguese document, at least until until someone steps up to support an European Portuguese one, and then we could maybe just have both, not that the differences matter too much.

**Comment by RockyTV** [comment]
If the agreement is to translate it to Brazilian Portuguese, I think the
file should be renamed to suit it better.

**Comment by ddevault** [comment]
There's another attempt here: #4931

**Pull request closed without merging**